### PR TITLE
Create dash.bernardonogueira8.is-a.dev.json

### PR DESCRIPTION
**Pull request by bernardonogueira8**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview
http://100.42.189.124:8000/

Hello, im is cool BD

### CODE REVIEW

**Comment by Samuraiflamesf** [comment]
New test

File: dash.bernardonogueira8.json
Comment (Stef-00012) [inline dash.bernardonogueira8.json:1]: this file should be in the domains folder

File: domains/cdn.bernardonogueira8.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Samuraiflamesf",
+    "email": "Samuraiflamesf@gmail.com",
+  },
+  "record": {
+    "A": ["100.42.189.124"]
```
Comment (Stef-00012) [inline domains/cdn.bernardonogueira8.json:7]: i cannot reach your website
> 404 page not found

File: dash.bernardonogueira8.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Samuraiflamesf",
+    "email": "Samuraiflamesf@gmail.com",
+  },
+  "record": {
+    "A": ["100.42.189.124"]
```
Comment (Stef-00012) [inline dash.bernardonogueira8.json:7]: i cannot reach your website
> 404 page not found

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: dash.bernardonogueira8.json
Comment (Samuraiflamesf) [reply dash.bernardonogueira8.json:1]: Thanks, I will solve it

**Pull request closed without merging**